cmd/oceantv/utils.go

```diff
@@ -37,6 +37,7 @@ import (
 	"strings"
 	"time"
 
+	googledatastore "cloud.google.com/go/datastore"
```

Comment:
Using "cloud.google.com/go/datastore" is breaking the encapsulation provided by openfish/datastore.
Please do this another way.

PS Apart from breaking the encapsulation, it breaks standalone mode.